docs/Contributing.mdx

```diff
@@ -11,5 +11,5 @@ So, from the bottom of our hearts, _thank you_ 💕.
 
 ## Getting Started
 
-Please read our [Code of Conduct](https://github.com/typescript-eslint/typescript-eslint/blob/main/CODE_OF_CONDUCT.md) first.
-If you're new to open source, you may also find the [How to Contribute to Open Source](https://opensource.guide/how-to-contribute) guide from https://opensource.guide useful.
+Please read our <u>[Code of Conduct](https://github.com/typescript-eslint/typescript-eslint/blob/main/CODE_OF_CONDUCT.md)</u> first.
+If you're new to open source, you may also find the <u>[How to Contribute to Open Source](https://opensource.guide/how-to-contribute)</u> guide from <u>https://opensource.guide</u> useful.
```

Comment:
[just asking] Oh, this approach would require us to remember to always wrap any links in our docs in a `<u>` tag. Maybe we should underline them with css only? `text-decoration: underline` should do the trick

Moreover, I suppose we might want links in blog posts to be underlined too. I think it would be much easier to maintain if the underline was added automatically

So I guess we can just apply `text-decoration` to all links inside blog/docs container. Or vice-versa - apply underline to all `a` tags on the page and disable `text-decoration` in undesirable places like the side bar or table of content

What do you think?

Reply:
Perhaps Docusaurus even has some functionality to customize markdown links (though I haven't searched)

Reply:
I definitely agree, I'm new to the MDX format and I was unsure how styling was applied in these pages in the first place aside from general markdown syntax.

Should be a fairly easy fix, I'll get it done in the morning.

Reply:
Yup, what @auvred said - we're definitely going to forget this if we need to do it manually. And it's a bit of a pain. There should be a straightforward path to get most or all of the links inside markdown content.